samples/react-native/webrequest/src/App.fs

```diff
@@ -0,0 +1,103 @@
+﻿module AwesomeApp
+
+open Fable.Import
+open Fable.Elmish
+open Fable.Import.Fetch
+open Fable.Helpers.Fetch
+open Fable.Core.Extensions
+open Fable.Extras.Promise
+open Fable.Extras.Cmd
+open Fable.Extras.PromiseBuilderImp
+
+// Model
+type Status =
+  | NotStarted
+  | InProgress
+  | Complete
+
+type Model =
+  { Url:string
+    StatusStr:string
+    Status:Status }
+
+type Msg =
+  | ChangeInput of string
+  | SendRequest
+  | Success of string
+  | Error of obj
+
+
+let init () =
+  { Url = "https://httpbin.org/ip"
+    StatusStr = "Request not yet sent"
+    Status = NotStarted }, []
+
+// Helpers
+let getUrl ( url : string ) =
+  promise {
+    let! res = GlobalFetch.fetch url
+    return! res.text()
+  }
+
+let statusString s =
+  match s with
+  | NotStarted -> "Request not yet started"
+  | InProgress -> "Request in progress"
+  | Complete -> "Request finished"
+
+// Update
+let update msg model : Model*Cmd<Msg> =
+  match msg with
+  | ChangeInput str ->
+    { model with Url = str}, []
+  | SendRequest ->
+    { model with
+        StatusStr = (statusString InProgress)
+        Status = InProgress }, Fable.Extras.Cmd.ofPromise (fun _ -> getUrl model.Url) Success Error
```

Comment:
As I started going beyond basics I realized names like `Fable.Extras` just don't make sense. Not sure `Fable` prefix adds a value either, one could use the core abstractions with any runtime.